contrib/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Usage
+#
+#   cd contrib
+#   docker build -t pyatv-mqtt-bridge .
+#   docker run -d \
+#       --restart=always \
+#       --name=pyatv-mqtt-bridge \
+#       -v /opt/pyatv-mqtt-bridge/config.json:/app/config.json:ro \
+#       pyatv-mqtt-bridge
+#
+
+FROM ubuntu
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+RUN apt-get install -y \
+        python3 python3-pip
+
+RUN apt-get install -y \
+        curl && \
+    curl -fsSL https://deb.nodesource.com/setup_14.x | bash - && \
+    apt-get install -y \
+        nodejs
+
+WORKDIR /app
+
+RUN pip3 install pyatv
+RUN npm install -g @sebbo2002/pyatv-mqtt-bridge
```

Comment:
I think I'll prefer to have the current version here. I would publish the image then on Docker Hub, so that woun't be an issue.

Reply:
I also see this pull request more as a "people need a Dockerfile". I didn't add one then because the script runs on a host that doesn't have Docker installed. Since I already use Semantic Release and I get nice versions out of it, I would build the Docker images with it as well. Similar to what I currently do with [ubud-server](https://github.com/ubud-app/server). It's a bit [more Bash](https://github.com/ubud-app/server/blob/develop/.github/workflows/release-publish.sh#L11), but works very well. 🤪